activerecord/test/activejob/destroy_async_job_not_present_test.rb

```diff
@@ -10,22 +10,36 @@ class UnusedBelongsTo < ActiveRecord::Base
   self.destroy_association_async_job = nil
 end
 
+class UndefinedConstantBelongsTo < ActiveRecord::Base
+  self.destroy_association_async_job = "UndefinedConstantJob"
+end
+
 class ActiveJobNotPresentTest < ActiveRecord::TestCase
   test "has_one dependent destroy_async requires activejob" do
```

Comment:
We might want to update these test names to match as well (since we're no longer really testing if `activejob` has been required).